### PR TITLE
feat(entity): get /entity return only osmId with text/plain header

### LINKED ISSUE #18: get only the osmId on get
solution:
add text/plain header to get the response as test with only the osmId

### PR DESCRIPTION
**Pull request by galta95**
| Question        | Answer |
| --------------- | ------ |
| New feature     | ✔ |

Closes #18

### CODE REVIEW

File: openapi3.yaml [hunk 1]
```diff
@@ -163,3 +166,6 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/Entity'
+    OsmId:
+      type: number
+      example: 1337
```
Comment (syncush) [inline openapi3.yaml:171]: defined twice

File: tests/integration/entity/entity.spec.ts [hunk 2]
```diff
@@ -159,6 +159,16 @@ describe('entity', function () {
         expect(response.body).toMatchObject(entity);
       });
 
+      it('should return 200 status code and only the osmId', async function () {
```
Comment (syncush) [inline tests/integration/entity/entity.spec.ts:162]: missing in description "with header of ....."

**Review by syncush — CHANGES_REQUESTED** [review]

File: openapi3.yaml (hunk 1 above)
Comment (galta95) [reply openapi3.yaml]: Done

File: tests/integration/entity/entity.spec.ts (hunk 2 above)
Comment (galta95) [reply tests/integration/entity/entity.spec.ts]: Done

**Review by syncush — CHANGES_REQUESTED** [review]
move `OsmId` definition to be one of the first schemas in the open API spec

**Review by syncush — APPROVED** [review]

**Pull request merged**